x-pack/test/alerting_api_integration/observability/helpers/alerting_api_helper.ts

```diff
@@ -64,5 +64,8 @@ export async function createRule<Params = ThresholdParams>({
       rule_type_id: ruleTypeId,
       actions,
     });
+  if (body.statusCode && body.statusCode !== 200) {
+    throw new Error(body.message);
```

Comment:
In case of an error, the message was not visible in logs, so I added this condition to surface the error message.